### PR TITLE
Mirillis.SplashUltimate version 2.7.0

### PR DESCRIPTION
**Pull request by IvanovXA0**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/16139)

### CODE REVIEW

File: manifests/m/Mirillis/SplashUltimate/2.7.0/Mirillis.SplashUltimate.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Mirillis.SplashUltimate
+PackageVersion: 2.7.0
+PackageLocale: en-US
+Publisher: Mirillis Ltd.
```
Comment (ItzLevvie) [inline manifests/m/Mirillis/SplashUltimate/2.7.0/Mirillis.SplashUltimate.locale.en-US.yaml:6]:
```suggestion
Publisher: Mirillis
```

File: manifests/m/Mirillis/SplashUltimate/2.7.0/Mirillis.SplashUltimate.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Mirillis.SplashUltimate
+PackageVersion: 2.7.0
+PackageLocale: en-US
+Publisher: Mirillis Ltd.
+PackageName: Splash Ultimate
```
Comment (ItzLevvie) [inline manifests/m/Mirillis/SplashUltimate/2.7.0/Mirillis.SplashUltimate.locale.en-US.yaml:7]:
```suggestion
PackageName: Splash
```

File: manifests/m/Mirillis/SplashUltimate/2.7.0/Mirillis.SplashUltimate.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: Mirillis.SplashUltimate
+PackageVersion: 2.7.0
+PackageLocale: en-US
+Publisher: Mirillis Ltd.
+PackageName: Splash Ultimate
+License: Copyright © 2009-2019 Mirillis Ltd.
+ShortDescription: Splash Player Ultimate
```
Comment (ItzLevvie) [inline manifests/m/Mirillis/SplashUltimate/2.7.0/Mirillis.SplashUltimate.locale.en-US.yaml:9]:
```suggestion
ShortDescription: The only player you will ever need. Stunning video quality thanks to advanced engine. Video conversion optimized for social media. Innovative and fully customizable user interface.
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Other than the changes I've made, everything looks good to me:
image

**Comment by IvanovXA0** [comment]
> Other than the changes I've made, everything looks good to me:
> image

Thank you for your comments and feedback.
I already made the recommended changes ;)

**Review by ItzLevvie — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**